.github/workflows/test.yaml

```diff
@@ -207,8 +207,10 @@ jobs:
         - centos-8-64
         - debian-9-64
         - debian-sid-64
+        # TODO: drop Fedora 32 once it goes EOL on 25.05.2021
```

Comment:
Let's add a google calendar reminder for this :)

Reply:
Added